src/core_modules/capture-core/components/WidgetErrorAndWarning/WidgetWarning/WidgetWarning.types.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+type rule = {
```

Comment:
NIT: I see that this rule has been reused between different widget, maybe we should declare a common type